app/templates/components/ui-table/cell/cell-simple-date.hbs

```diff
@@ -4,6 +4,6 @@
   </span>
 {{else if this.record}}
   <span>
-    {{general-date this.record (if this.props.options.dateFormat this.props.options.dateFormat 'date-time-short')}}
+    {{general-date this.record 'date-time-short'}}
```

Comment:
This should work with the passed format as well

Reply:
the format being passed was leading to code duplication bcz for the same format we have key-word so I taught we can do like this?

Reply:
You can leave that without duplication and still allow custom format in this component